binder/requirements.txt

```diff
@@ -1,5 +1,5 @@
 -r ../requirements/build.txt
 -r ../requirements/install.txt
 -r ../requirements/extras.txt
+.
```

Comment:
```suggestion

```

What is this?  It doesn't look right to me.

Reply:
That's a good question!  The period is to say that binder should install the version of PlasmaPy in the repository, and not the official release.  This means we'll open the notebooks with the version of PlasmaPy in the `main` branch, and not the latest release.